omero/autogen_docs

```diff
@@ -33,10 +33,14 @@ echo "Generating DB script example"
 (cd $WORKSPACE && OMERO.server/bin/omero db script --password secretpassword 2>&1) | sed "s|$WORKSPACE|/home/omero|" > omero/downloads/cli/db-script-example.txt
 
 echo "Generating Web configuration templates"
-$WORKSPACE/OMERO.server/bin/omero web config nginx | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/nginx-omero.conf
-$WORKSPACE/OMERO.server/bin/omero web config apache | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/apache-omero.conf
-$WORKSPACE/OMERO.server/bin/omero web config apache-fcgi | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/apache-fcgi-omero.conf
-
+$WORKSPACE/OMERO.server/bin/omero web config nginx | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/install-web/nginx-omero.conf
+$WORKSPACE/OMERO.server/bin/omero web config apache | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/install-web/apache-omero.conf
+$WORKSPACE/OMERO.server/bin/omero web config apache-fcgi | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/install-web/apache-fcgi-omero.conf
+$WORKSPACE/OMERO.server/bin/omero config set omero.web.application_server "wsgi"
+$WORKSPACE/OMERO.server/bin/omero web config apache-wsgi | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/unix/install-web/apache-wsgi-omero.conf
```

Comment:
That content is autogenerated, these files are only needed to build become green, see http://www.openmicroscopy.org/site/support/omero5.1-staging/sysadmins/unix/install-web/install-wsgi.html